app/assets/javascripts/discourse/app/components/sidebar/hamburger-dropdown.gjs

```diff
@@ -20,6 +21,16 @@ export default class SidebarHamburgerDropdown extends Component {
     this.appEvents.trigger("sidebar-hamburger-dropdown:rendered");
   }
 
+  @action
+  focusFirstLink() {
+    next(() => {
+      const firstLink = document.querySelector(".sidebar-hamburger-dropdown a");
+      if (firstLink) {
+        firstLink.focus();
+      }
+    });
```

Comment:
Generally speaking I prefer to wrap DOM manipulation in schedule("afterRender") than next(). But in this case I don't think you need it, did you try without?